libraries/Microsoft.Bot.Builder.Dialogs/Prompts/PromptOptions.cs

```diff
@@ -51,5 +52,16 @@ public class PromptOptions
         /// </summary>
         /// <value>Additional options for use with a prompt validator.</value>
         public object Validations { get; set; }
+
+        /// <summary>
+        /// Gets or sets any extra properties to include in the results.
+        /// </summary>
+        /// <value>
+        /// Any extra properties to include in the results.
+        /// </value>
+        [JsonExtensionData(ReadData = true, WriteData = true)]
+#pragma warning disable CA2227 // Collection properties should be read only  (we can't change this without breaking binary compat)

```

Comment:
I don't think we need the setter, this is a new property so we wouldn't be breaking binary compat, we can probably remove the pragma and make the property readonly

Reply:
I tried, unsuccessfully, to remove this pragma.